docs/src/main/paradox/pravega.md

```diff
@@ -0,0 +1,115 @@
+# Pravega
+
+[Pravega](http://www.pravega.io/) provides a new storage abstraction - a stream - for continuous and unbounded data. A Pravega stream is a durable, elastic, append-only, unbounded sequence of bytes that has good performance and strong consistency.
```

Comment:
A Pravega stream is not a single unbounded sequence as it can have parallel segments. He is a possible rephrase:

> A Pravega stream is an elastic set of durable and append-only segments, each segment being an unbounded sequence of bytes. Streams provide exactly-once semantics, and atomicity for groups of events using transactions.